examples/contracts/user_storage.py

```diff
@@ -1,20 +1,25 @@
-from backend.node.genvm.icontract import IContract
+# { "Depends": "py-genlayer:test" }
```

Comment:
@kp2pml30 is this just a comment or is this something that the GenVM uses?

Reply:
Something genvm uses

Reply:
Ok so we need to add this line on some contracts. Do we have some documentation for general Intelligent Contracts implementation?

Reply:
No we don't have it in any public repos. Where should I put it?

Reply:
There was a repo for "Marvin" documentation: https://github.com/yeagerai/marvin but right now I think that just a Google Doc with the basic stuff would be great to explain that to other devs that are using the simulator today :)

Reply:
Putting docs in private repos is sort of useless...

Reply:
of course, but they aren't if we release them at some point. 
On the other hand, at least the internal team should be able to understand how to write intelligent contracts fully.

Reply:
For example this line `# { "Depends": "py-genlayer:test" }` how can I know when and how to use it?